### DISCUSSION TITLE
Are sub namespaces of System allowed?

### DISCUSSION THREAD

**Discussion by outercloudstudio** [SebLague/Chess-Challenge#212] [Q&A]
^^^^

**Comment by braftery77** [comment]
I do not think so other than the ones specified. Otherwise there would have been no need for the specifications on the other subnamespaces

**Comment by SebLague** [comment]
No, only the ones explicitly listed are allowed

**Reply by WhiteMouse1** [reply]
What? Not even System.Stockfish?!
